### PR TITLE
add: `hyfetch-git`

### PR DESCRIPTION
**Pull request by pythoncrazy**
Hi, I have added hyfetch-git, based upon https://github.com/hykilpikonna/hyfetch/ & https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=hyfetch-git

### CODE REVIEW

File: hyfetch-git/hyfetch-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,22 @@
+pkgname="hyfetch-git"
+gives="hyfetch"
+pkgdesc="Neofetch with LGBTQ pride flags."
```
Comment (Elsie19) [inline hyfetch-git/hyfetch-git.pacscript:3]:
```suggestion
pkgdesc="Neofetch with LGBTQ pride flags"
```

File: hyfetch-git/hyfetch-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+makedepends=('python3-pip')
+external_connection="true"
+
+build() {
+  cd "${_archive}"
+  pip install .
+}
+
+package() {
+  cd "${_archive}"
```
Comment (Elsie19) [inline hyfetch-git/hyfetch-git.pacscript:18]: Do not use `${_archive}` in new scripts.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: hyfetch-git/hyfetch-git.pacscript (hunk 2 above)
Comment (pythoncrazy) [reply hyfetch-git/hyfetch-git.pacscript:18]: What is the modern alternative? Is it `cd "${srcdir}/${pkgname%-git}"`? Thank you for all of your wonderful help and support!

Comment (Elsie19) [reply hyfetch-git/hyfetch-git.pacscript:18]: Well first of all, it should be `${pkgname%-git}`, but I would suggest just not making source download without the git, so then you wouldn't have to have `${pkgname%-git}` everywhere, and just have `${pkgname}`.

**Review by oklopfer — APPROVED** [review]

**Pull request merged**